example/lib/device_details/device_details_bloc.dart

```diff
@@ -45,7 +45,8 @@ class DeviceDetailsBloc {
     _logsController = PublishSubject<List<DebugLog>>();
 
     log = (text) {
-      _logs.insert(0, DebugLog(DateTime.now().toString(), text));
+      var now = DateTime.now();
+      _logs.insert(0, DebugLog('${now.hour}:${now.minute}:${now.second}:${now.millisecond}', text));
```

Comment:
Milliseconds are separated by `.`, not `:`.
${now.hour}:${now.minute}:${now.second}.${now.millisecond}